src/clients/c/tb_client/context.zig

```diff
@@ -163,6 +166,7 @@ pub fn ContextType(
             InvalidDataSize,
         };
 
+        gpa: GPA,
```

Comment:
Do we need both `gpa` and `allocator`?
I think we can store just the GPA reference here and use `self.gpa.allocator()`.